src/components/RangeSlider/index.js

```diff
@@ -16,13 +16,28 @@ class RangeSlider extends Component {
     }
   }
 
+  roundLabelValue = value => {
+    const {
+      incrementSize
+    } = this.props
+
+    if (!isFinite(incrementSize) || incrementSize <= 0) return value;
+    var e = 1, p = 0;
+    while (Math.round(incrementSize * e) / e !== incrementSize) { e *= 10; p++; }
```

Comment:
Would this be more readable if we expanded the variable names to be more indicative of the purpose?

For example:
* `e` could be `multiplier`
* `p` could be `precision`

```suggestion
    let multiplier = 1
    let precision = 0
    while (Math.round(incrementSize * multiplier) / multiplier !== incrementSize) {
        multiplier *= 10
        precision += 1
    }
```

A further improvement to readability could be pulling this particular algorithm out to a named function.

Reply:
@tobyjsullivan 

I added a comment above ^ code block with a description of what it's doing:
```
// precision function => obtains number of digits after decimal point of incrementSize
if (!isFinite(incrementSize) || incrementSize <= 0) return value;
let multiplier = 1, precision = 0;
while (Math.round(incrementSize * multiplier) / multiplier !== incrementSize) { 
  multiplier *= 10; precision++; 
}
```

I separated these into two different functions but it made the code a bit more complicated (because of the first `if` statement mostly) so I just went with this. If you still think it would be improved by pulling out the function, I will implement that! Thanks Toby